### DISCUSSION TITLE
What is "Archive Extract Folder" & it's functionality ?

### DISCUSSION THREAD

**Discussion by shuvashish76** [TeamAmaze/AmazeFileManager#3339] [Q&A]
Settings > Behaviour > Archive Extract Folder (AEF)
> Archive files will be extracted to this folder. Default value is the directory in which archive is present.

---

Question 1 : I changed AEF to some specific folder, compressed some files as `.zip` format then extracted them. It extracted files to in which the the archive is present not to my specified AEF.
So AEF functionality is confusing to me.

Question 2 : _Assuming current AEF behavior is a bug & it should extract files to user specified folder._ So you picked your folder & all archive files extracted to that particular folder only, but let's say after a while you changed your mind & now you wanna _extract Archive files to the directory in which archive is present. How to revert that thing?

(Just to make it clear : _Compress_ is the process & _Archive_ is its result i.e the compressed file. CMIIW)

**Comment by VishnuSanal** [comment]
This feature feels unnecessary to me. The location where new ZIPs were placed was removed with #2393 (#2381). And those were defaulted out to the present working directory. I personally feel like a user expects archives to be extracted on the same folder where the archive is present.
